### PR TITLE
table summary of layergroup mode functionality

### PR DESCRIPTION
**Pull request by jodygarnett**

### CODE REVIEW

File: doc/en/user/source/data/webadmin/layergroups.rst [hunk 1]
```diff
@@ -19,9 +23,20 @@ Layer group behaviour can be configured by setting its :guilabel:`mode`. There a
 
 If a layer is included in any non *single* mode group, it will no longer be listed in the flat layer list.  It will still be possible to include the layer in other layer groups. 
 
-.. figure:: img/data_layergroups.png
++------------------------+-------+-------------------+----------------+------------------+
```
Comment (bmmpxf) [inline doc/en/user/source/data/webadmin/layergroups.rst]: Not the hugest fan of visual tables like this, as I think they can be cumbersome to edit (realigning columns). I prefer list-tables personally. But the table is definitely appreciated.

**Comment by bmmpxf** [comment]
All good here.

**Comment by jodygarnett** [comment]
Thanks docs applied to master; only applying to master as this table documents a new feature (opaque container layergroup mode).

**Comment by bmmpxf** [comment]
Thank you!

**Comment by bmmpxf** [comment]
Hopefully the new layer group mode won't be "opaque" to people. ;)

**Pull request merged**